Number_types/include/CGAL/GMP/Gmpzf_type.h

```diff
@@ -422,7 +422,7 @@ std::pair<std::pair<double, double>, long> Gmpzf::to_interval_exp() const
   // get surrounding interval of the form [l * 2 ^ k, u * 2^ k]
   // first get mantissa in the form l*2^k, with 0.5 <= d < 1;
   // truncation is guaranteed to go towards zero
-  long k = 0;
+  Exponent k = 0;
   double l = mpz_get_d_2exp (&k, man());
```

Comment:
Well, the correct type for k is whatever `mpz_get_d_2exp` is documented to take as first argument (minus the pointer). According to the GMP documentation, `long` is right, and the reverse patch in function `to_double` would make sense.